.evergreen/config.yml

```diff
@@ -697,45 +723,45 @@ buildvariants:
 - matrix_name: "driver-min"
   matrix_spec:
     driver: "min"
-    ruby: "ruby-2.3"
+    ruby: ["ruby-2.3","ruby-2.4", "ruby-2.5"]
     mongodb-version: "3.6"
     topology: "standalone"
   display_name: "${ruby}, ${driver}, ${mongodb-version}, ${topology}"
   run_on:
     - rhel70-small
   tasks:
      - name: "test"
-- matrix_name: "rails-5.1"
+- matrix_name: "rails-5.2"
   matrix_spec:
     driver: "current"
-    ruby: "ruby-2.4"
+    ruby: ["ruby-2.6", "ruby-2.7"]
     mongodb-version: "4.2"
     topology: "standalone"
-    rails: '5.1'
-  display_name: "${rails}, ${driver}, ${mongodb-version}"
+    rails: '5.2'
+  display_name: "${ruby}, ${driver}, ${mongodb-version}, ${topology}, ${rails}"
   run_on:
     - rhel70-small
   tasks:
      - name: "test"
-- matrix_name: "rails-5.2"
+- matrix_name: "rails-6.0"
   matrix_spec:
     driver: "current"
-    ruby: "ruby-2.6"
-    mongodb-version: "4.2"
+    ruby: ["ruby-2.6", "ruby-2.7", "ruby-3.0"]
+    mongodb-version: "4.4"
     topology: "standalone"
-    rails: '5.2'
-  display_name: "${rails}, ${driver}, ${mongodb-version}"
+    rails: '6.0'
+  display_name: "${ruby}, ${driver}, ${mongodb-version}, ${topology}, ${rails}"
   run_on:
     - rhel70-small
   tasks:
      - name: "test"
-- matrix_name: "rails-6.0"
+- matrix_name: "rails-6.1"
```

Comment:
This shouldn't be needed either. The default rails should be the most recent one allowed by the gemspec, which currently is 6.1. We have gemfiles for older versions, meaning 6.0 through 5.1.

Reply:
I'll bring back gemfile and configuration for 5.1